### PR TITLE
seperated classes

### PR DESCRIPTION
**Pull request by martirosyan94**
Seperate class

### CODE REVIEW

File: ResgistrationSystem/ResgistrationSystem/UserMenegment.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    {
+        public static List<User> Users { get; set; } = new();
+        private int command;
+        private string fileName = "account_info.json";
+        public User CurrentUser { get; private set; }
+        public bool IsLogined { get; private set; } = false;
+        public UserMenegment()
+        {
+            if (!File.Exists(fileName))
+                File.Create(fileName);
```
Comment (mruben0) [inline ResgistrationSystem/ResgistrationSystem/UserMenegment.cs:22]: Always use brackets with if statement

File: ResgistrationSystem/ResgistrationSystem/UserMenegment.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            string surName = Console.ReadLine();
+            Console.WriteLine("age");
+            int age = int.Parse(Console.ReadLine());
+            Console.WriteLine("email");
+            string email = Console.ReadLine();
+            Console.WriteLine("passwrod");
+            string password = Console.ReadLine();
+
+            string userInfo = JsonSerializer.Serialize(new User(name, surName, age, email, password));
+            StreamWriter sw = File.AppendText(fileName);
```
Comment (mruben0) [inline ResgistrationSystem/ResgistrationSystem/UserMenegment.cs:40]: use streamwriter with using statement to dispose.
[Example](https://docs.microsoft.com/en-us/dotnet/api/system.io.streamwriter?view=net-5.0)

File: ResgistrationSystem/ResgistrationSystem/UserMenegment.cs [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                {
+                    Console.WriteLine("You have successfully logged in");
+                    IsLogined = true;
+                    return;
+                }
+            }    
+               
+            // Todo firstordefault first single singleordefault (read, define best one)
+ /*           CurrentUser = Users.FirstOrDefault(e => e.Email == userName && e.Password == password);
+            if (CurrentUser is not null)
```
Comment (mruben0) [inline ResgistrationSystem/ResgistrationSystem/UserMenegment.cs:66]: Avoid pushing commented code

File: ResgistrationSystem/ResgistrationSystem/UserMenegment.cs (hunk 1 above)
Comment (martirosyan94) [reply ResgistrationSystem/ResgistrationSystem/UserMenegment.cs:22]: fixed

File: ResgistrationSystem/ResgistrationSystem/UserMenegment.cs (hunk 2 above)
Comment (martirosyan94) [reply ResgistrationSystem/ResgistrationSystem/UserMenegment.cs:40]: Do you mean to include it in try/catch?
Like this:
StreamWriter c =null;
try
{
  c = new StreamWriter(fileFullPath, true);
  c.WriteLine("hello");
}
finally
{
  if (c!= null)
      c.Close();
}

**Pull request merged**